### PR TITLE
csi 4057 add settings files

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: controllers/tests/controller_server/addons_server_test.py [hunk 1]
```diff
@@ -21,30 +27,37 @@ def tested_method_response_class(self):
...
+        self.mediator.client = Mock()
+
+        self.storage_agent = MagicMock()
+        mock_get_agent(self, ADDON_SERVER_PATH)
+
+        self.request = ProtoBufMock()
+        self.request.secrets = {"username": SECRET_USERNAME_VALUE, "password": SECRET_PASSWORD_VALUE,
+                                "management_address": SECRET_MANAGEMENT_ADDRESS_VALUE}
+        self.request.volume_id = "{}:{};{}".format("A9000", OBJECT_INTERNAL_ID, VOLUME_UID)
+        self.request.replication_id = "{}:{};{}".format("A9000", OTHER_OBJECT_INTERNAL_ID, VOLUME_UID)
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply controllers/tests/controller_server/addons_server_test.py:41]: the ticket is for mediators

File: controllers/tests/array_action/ds8k/test_array_mediator_ds8k.py [hunk 2]
```diff
@@ -12,105 +12,123 @@
...
+    GET_HOST_MAPPINGS_ID_ATTR_KEY, GET_HOST_MAPPINGS_VOLUME_ATTR_KEY, GET_HOST_LOGIN_PORTS_ATTR_KEY, \
+    LOGIN_PORT_STATE_OFFLINE, GET_HOSTS_WWPN_ATTR_KEY, \
+    GET_HOSTS_HOST_PORTS_BRIEFS_ATTR_KEY
+from controllers.tests.array_action.test_settings import DUMMY_ERROR_MESSAGE, DUMMY_HOST_NAME1, DUMMY_FC_WWN1, \
+    DUMMY_FC_WWN2, FC_CONNECTIVITY_TYPE, DUMMY_CONNECTIVITY_TYPE, DUMMY_FC_WWN3, DUMMY_FC_WWN4, VOLUME_NAME_ATTR_KEY, \
+    VOLUME_ID_ATTR_KEY, DUMMY_LUN_ID
+from controllers.tests.common.test_settings import VOLUME_NAME, DUMMY_POOL1, SECRET_USERNAME_VALUE, \
+    SECRET_PASSWORD_VALUE, VOLUME_UID, \
+    SNAPSHOT_NAME, SNAPSHOT_VOLUME_UID, INTERNAL_SNAPSHOT_ID, VOLUME_OBJECT_TYPE, SNAPSHOT_OBJECT_TYPE, HOST_NAME, \
+    SOURCE_VOLUME_NAME, DUMMY_POOL2, SOURCE_VOLUME_ID
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply controllers/tests/array_action/ds8k/test_array_mediator_ds8k.py]: done

File: controllers/tests/array_action/svc/array_mediator_svc_test.py [hunk 3]
```diff
@@ -97,37 +125,40 @@ def _test_mediator_method_client_cli_failure_error(self, mediator_method, args,
...
     def test_get_volume_with_source_success(self):
-        volume = self._test_get_volume({'vdisk_uid': "source_id", 'fc_id': '1'})
-        self.assertEqual("source_id", volume.source_id)
+        volume = self._test_get_volume({"vdisk_uid": SOURCE_VOLUME_ID, "fc_id": svc_settings.DUMMY_INTERNAL_ID1})
+        self.assertEqual(SOURCE_VOLUME_ID, volume.source_id)
 
     def test_get_volume_with_source_and_flashcopy_enabled(self):
-        volume = self._test_get_volume({'vdisk_uid': "source_id", 'fc_id': '1'}, is_virt_snap_func=True,
+        volume = self._test_get_volume({"vdisk_uid": SOURCE_VOLUME_ID, "fc_id": svc_settings.DUMMY_INTERNAL_ID1},
+                                       is_virt_snap_func=True,
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply controllers/tests/array_action/svc/array_mediator_svc_test.py]: done

File: controllers/tests/array_action/svc/array_mediator_svc_test.py [hunk 4]
```diff
@@ -1246,319 +1360,366 @@ def _test_get_volume_mappings_lsvdisk_cli_failure_error(self, volume_name, error
...
+                                           svc_settings.INVALID_NAME_1)
+        self._expand_volume_lsvdisk_errors(CLIFailureError("CMMVC5741E"), array_errors.InvalidArgumentError,
+                                           svc_settings.INVALID_NAME_SYMBOLS)
+        self._expand_volume_lsvdisk_errors(CLIFailureError(DUMMY_ERROR_MESSAGE), CLIFailureError)
+        self._expand_volume_lsvdisk_errors(Exception(DUMMY_ERROR_MESSAGE), Exception)
 
     def test_create_host_nvme_success(self):
-        self.svc.create_host("host_name", Initiators(['Test_nqn'], ['wwn1', 'WWN2'], ['iqn.test.s1']), "")
-        self.svc.client.svctask.mkhost.assert_called_once_with(name='host_name', nqn='Test_nqn', protocol='nvme')
+        self.svc.create_host(HOST_NAME,
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply controllers/tests/array_action/svc/array_mediator_svc_test.py]: done

**Review by matancarmeli7 — DISMISSED** [review]
LGTM

**Review by ELENAGER — APPROVED** [review]

**Review by matancarmeli7 — APPROVED** [review]

**Pull request merged**